.github/workflows/hrm-ecs-ci-multiregion.yml

```diff
@@ -22,10 +22,10 @@
 # 4. Store an IAM user access key in GitHub Actions secrets named `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
 #    See the documentation for each action used below for the recommended IAM policies for this IAM user,
 #    and best practices on handling the access key credentials.
-name: "[Multi Region] Deploy HRM to Amazon ECS"
+name: '[Multi Region] Deploy HRM to Amazon ECS'
```

Comment:
This can probably be renamed to simply 'Deploy HRM to Amazon ECS' - the [Multi Region] qualifier was only there for when we still had the legacy workflow with a hardcoded region availalbe

Reply:
I haven't actually made any changes to the hrm deployment yet. This was my IDE applying it's yaml lint rules on save (oops). I can revert these changes if needed.